docker/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3'
+services:
+  rabbit:
+    image: rabbitmq
+    container_name: video-server-rabbitmq
+    environment:
+      - RABBITMQ_DEFAULT_USER=guest
+      - RABBITMQ_DEFAULT_PASS=guest
+  mongodb:
+    image: mongo:3
+    container_name: video-server-mongodb
+    volumes:
+      - mongodb_data:/data/db
+  server:
+    build: .
+    command: honcho start
+    container_name: video-server
+    ports:
+      - "5050:5050"
+    depends_on:
+      - rabbit
+      - mongodb
+    links:
+      - rabbit
+      - mongodb
+    environment:
+      - MEDIA_STORAGE=filesystem
+      - FS_MEDIA_STORAGE_PATH=/opt/media
+      - RABBIT_MQ_URL=pyamqp://guest@rabbit//
+      - MONGO_HOST=mongodb
+    volumes:
+      - server_data:/data/db:rw
+volumes:
+  mongodb_data: {}
+  server_data: {}
```

Comment:
For now video/thumbnail files are stored in named volume. If needed it's possible to explore these files by entering a container using `exec -it`.
As an option, I can mount a host path for it.